### ISSUE TITLE
Full documentation missing from CloudFront event type

### ISSUE THREAD

**Issue by trondhindenes** [serverless/serverless#6785]
# Bug Report

## Description
The documentation for cloudfront is very thin (https://serverless.com/framework/docs/providers/aws/events/cloudfront/). There's a few examples, but there's no complete documentation of all supported params. Since its likely that users have to use a `Resources` object to describe the Cloudfront distribution in detail, this even type is potentially quite complex to get right. It would be good of the documentation was a bit richer.

**Comment by micheleangioni** [comment]
I agree with @trondhindenes . More documentation would be very appreciated.

**Comment by BenCoffeed** [comment]
Very thin indeed. Also, is it currently possible to assign ACM certs for SSL support of distribution?
